frontends/web/src/app.tsx

```diff
@@ -81,30 +81,30 @@ class App extends Component<Props, State> {
         this.onBitBoxBasesDetectedChanged();
         this.unsubscribe = apiWebsocket(({ type, data, meta }) => {
             switch (type) {
-            case 'backend':
-                switch (data) {
-                case 'newTxs':
-                    apiPost('notify-user', {
-                        text: this.props.t('notification.newTxs', {
-                            count: meta.count,
-                            accountName: meta.accountName,
-                        }),
-                    });
+                case 'backend':
```

Comment:
is there a way to make the indent rule not indent `case`? imho it worsens readability. not a big deal if not.

Reply:
Half of the switch case were already indented, I do not agree that this worsens readability, indenting usually helps visualizing where the block starts/ends. But if this is your personal preference then I don't mind, but it is very common in js world

- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch
- https://2ality.com/2020/02/typescript-exhaustiveness-checks-via-exceptions.html
- (expect Crockford, he doesn't indent either :P https://www.crockford.com/code.html#switch%20statement)

Reply:
Interesting that the first two examples have 2 space indents, not 4 space, so the code is not indented 8 spaces in, but just 4. I wouldn't mind going to 2 spaces either :D 

Anyway, I also don't have a very strong opinion. Since you spend the most time in the frontend, it should fit your style :sunglasses: